crates/wasm/wasm-chain-components/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "hermes-wasm-chain-components"
+version = { workspace = true }
+edition = { workspace = true }
+license = { workspace = true }
+keywords = { workspace = true }
+repository = { workspace = true }
+authors = { workspace = true }
+rust-version = { workspace = true }
+description = """
+    Context-generic chain components related to CosmWasm
+"""
```

Comment:
description should be single line.

https://doc.rust-lang.org/cargo/reference/manifest.html#the-description-field

Reply:
Also, because of the multiline, `taplo` couldn't format this block.

Reply:
833857c4b4a16275966eae81a768bec6d4e7e0af